lib/AST/SwiftNameTranslation.cpp

```diff
@@ -167,6 +169,11 @@ swift::cxx_translation::getNameForCxx(const ValueDecl *VD,
   if (isa<ConstructorDecl>(VD))
     return "init";
 
+  if (VD->isOperator()) {
+    std::string name = ("operator" + VD->getBaseIdentifier().str()).str();
```

Comment:
I think this should use llvm::Twine + llvm::StringRef because that won't allocate as much as `getIdentifier` already accepts a StringRef so strings don't need to materialized eagerly.

Reply:
Is there a way to convert a `Twine` to a `StringRef` without copying the strings? I thought we'd still need to make a copy of the strings.